docs/integrations/builtin/app-nodes/n8n-nodes-base.oracledb/index.md

```diff
@@ -0,0 +1,235 @@
+---
+title: Oracle Database node documentation
+description: Learn how to use the Oracle Database node in n8n. Follow technical documentation to integrate Oracle Database node into your workflows.
+contentType: [integration, reference]
+priority: critical
+---
+
+# Oracle Database node
+
+Use the Oracle Database node to automate work in Oracle Database, and integrate Oracle Database with other applications. n8n has built-in support for a wide range of Oracle Database features, including executing an SQL query, as well as inserting, and updating rows in a database.
+
+On this page, you'll find a list of operations the Oracle Database node supports and links to more resources.
+
+/// note | Credentials
+Refer to [Oracle Database credentials](/integrations/builtin/credentials/oracledb.md) for guidance on setting up authentication.
+///
+
+--8<-- "/_snippets/integrations/builtin/app-nodes/ai-tools.md"
+
+## Operations
+
+* [**Delete**](#delete): Delete an entire table or rows in a table
+* [**Execute SQL**](#execute-sql): Execute an SQL statement
+* [**Insert**](#insert): Insert rows in a table
+* [**Insert or Update**](#insert-or-update): Insert or update rows in a table
+* [**Select**](#select): Select rows from a table
+* [**Update**](#update): Update rows in a table
+
+### Delete
+
+Use this operation to delete an entire table or rows in a table.
+
+Enter these parameters:
+
+- **Credential to connect with**: Create or select an existing [Oracle Database credential](/integrations/builtin/credentials/oracledb.md).
+- **Operation**: Select **Delete**.
+- **Schema**: Choose the schema that contains the table you want to work on. Select **From list** to choose the schema from the dropdown list or **By Name** to enter the schema name.
+- **Table**: Choose the table that you want to work on. Select **From list** to choose the table from the dropdown list or **By Name** to enter the table name.
+- **Command**: The deletion action to take:
+	- **Truncate**: Removes the table's data but preserves the table's structure.
+	- **Delete**: Delete the rows that match the "Select Rows" condition. If you don't select anything, Oracle Database deletes all rows.
+		- **Select Rows**: Define a **Column**, **Operator**, and **Value** to match rows on. The value
+	can be passed as JSON using expression or string.
+		- **Combine Conditions**: How to combine the conditions in "Select Rows". **AND** requires all conditions to be true, while **OR** requires at least one condition to be true.
+	- **Drop**: Deletes the table's data and structure permanently.
+
+#### Delete options
+
+- **Auto Commit**: Whether this property is true, then the transaction in the current connection is automatically committed at the end of statement execution.
```

Comment:
When this property is set to *true*, the transaction in the current connection is automatically committed at the end of statement execution.